composer/profiler/profiler.py

```diff
@@ -214,11 +224,11 @@ def get_action(self, batch_idx: int) -> ProfilerAction:
         return ProfilerAction.ACTIVE
 
     @property
-    def event_handlers(self):
+    def handlers(self):
         """Profiler event handlers."""
-        return self._event_handlers
+        return self._event_handlers + self._profilers
 
-    def merge_traces(self):
+    def merge_traces(self, callbacks: Callback):
```

Comment:
If `state` is being passed in on `__init__`, then the callbacks need not be passed in here.